### PR TITLE
Feature/id 402/change text field in the search

### PR DESCRIPTION
**Pull request by Karvlox**
## What?
The functionality of being able to change the SearchView for a TextView was added so that the results of a search can be observed. The compass icon was also redirected so that it does not overlap over another icon.

## Why?
So that we have a better aesthetic and pleasant for the user.

## How?
The activityMain was modified and the activityMain.xml was also modified.

## Trainer
@mmoralesv 

## Code reviewers
@saletG08 
@VillalobosGarciaGaby 
@aguirrekarina

### CODE REVIEW

**Comment by aguirrekarina** [comment]
the map customization is displayed somewhere else and not where it should be.
image

File: app/src/main/res/drawable/searchview_rounded_bg.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <solid android:color="#ffffff" /> <!-- Cambia el color de fondo si lo deseas -->
```
Comment (VillalobosGarciaGaby) [inline app/src/main/res/drawable/searchview_rounded_bg.xml]: delete these comments.

File: app/src/main/res/drawable/searchview_rounded_bg.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+    <corners android:radius="20dp" /> <!-- Cambia el radio para ajustar el nivel de redondeo -->
```
Comment (VillalobosGarciaGaby) [inline app/src/main/res/drawable/searchview_rounded_bg.xml]: delete these comments.

File: app/src/main/java/com/isc/hermes/SearchViewActivity.java [hunk 3]
```diff
@@ -97,6 +98,12 @@ public void onItemClick(WayPoint wayPoint) {
         editor.putFloat("longitude", (float)wayPoint.getLongitude());
         editor.apply();
         finish();
+        /* Optional
```
Comment (VillalobosGarciaGaby) [inline app/src/main/java/com/isc/hermes/SearchViewActivity.java]: delete these comments or if it works for you later add a TODO

File: app/src/main/java/com/isc/hermes/MainActivity.java [hunk 4]
```diff
@@ -365,9 +373,25 @@ public void changeMapStyle(View view) {
      * The markers are added using the MarkerManager instance.
      */
     private void addMarkers() {
-        markerManager.addMarkerToMap(mapView, sharedSearcherPreferencesManager.getPlaceName(),
-                sharedSearcherPreferencesManager.getLatitude(),
-                sharedSearcherPreferencesManager.getLongitude());
+
+
+        // Obtiene la lista de lugares y sus coordenadas
```
Comment (VillalobosGarciaGaby) [inline app/src/main/java/com/isc/hermes/MainActivity.java]: delete this comment

File: app/src/main/java/com/isc/hermes/MainActivity.java [hunk 5]
```diff
@@ -365,9 +373,25 @@ public void changeMapStyle(View view) {
...
+        List<String> placeNames = Collections.singletonList(sharedSearcherPreferencesManager.getPlaceName());
+        List<Double> latitudes = Collections.singletonList(sharedSearcherPreferencesManager.getLatitude());
+        List<Double> longitudes = Collections.singletonList(sharedSearcherPreferencesManager.getLongitude());
+
+        // Agrega las nuevas marcas al mapa y actualiza el texto en searchView
```
Comment (VillalobosGarciaGaby) [inline app/src/main/java/com/isc/hermes/MainActivity.java]: delete this comment

File: app/src/main/java/com/isc/hermes/MainActivity.java [hunk 6]
```diff
@@ -187,6 +185,17 @@ private void changeSearchView() {
         });
     }
 
+    /* Review
```
Comment (VillalobosGarciaGaby) [inline app/src/main/java/com/isc/hermes/MainActivity.java]: delete these comments or if it works for you later add a TODO

File: app/src/main/res/drawable/searchview_rounded_bg.xml (hunk 1 above)
Comment (Karvlox) [reply app/src/main/res/drawable/searchview_rounded_bg.xml]: That is not my correspondence.

File: app/src/main/java/com/isc/hermes/SearchViewActivity.java (hunk 3 above)
Comment (Karvlox) [reply app/src/main/java/com/isc/hermes/SearchViewActivity.java]: Done.

File: app/src/main/java/com/isc/hermes/MainActivity.java (hunk 5 above)
Comment (Karvlox) [reply app/src/main/java/com/isc/hermes/MainActivity.java]: That is not my correspondence.

File: app/src/main/java/com/isc/hermes/MainActivity.java (hunk 6 above)
Comment (Karvlox) [reply app/src/main/java/com/isc/hermes/MainActivity.java]: Done.

File: app/src/main/java/com/isc/hermes/MainActivity.java (hunk 4 above)
Comment (Karvlox) [reply app/src/main/java/com/isc/hermes/MainActivity.java]: Done.

**Review by aguirrekarina — APPROVED** [review]

File: app/src/main/res/drawable/searchview_rounded_bg.xml (hunk 2 above)
Comment (Karvlox) [reply app/src/main/res/drawable/searchview_rounded_bg.xml]: That does not apply to me

File: app/src/main/res/drawable/searchview_rounded_bg.xml (hunk 1 above)
Comment (VillalobosGarciaGaby) [reply app/src/main/res/drawable/searchview_rounded_bg.xml]: but it should not have a comment in a language other than Spanish.

File: app/src/main/res/drawable/searchview_rounded_bg.xml (hunk 2 above)
Comment (VillalobosGarciaGaby) [inline app/src/main/res/drawable/searchview_rounded_bg.xml]: delete these comments.

**Review by VillalobosGarciaGaby — APPROVED** [review]

**Review by saletG08 — APPROVED** [review]
I don't have any comment

**Pull request merged**